ctapipe/io/tests/test_dl1writer.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
+
+from ctapipe.io.dl1writer import DL1Writer, DL1_DATA_MODEL_VERSION
+from ctapipe.utils import get_dataset_path
+from ctapipe.io import event_source
+from ctapipe.calib import CameraCalibrator
+from pathlib import Path
+from ctapipe.instrument import SubarrayDescription
+import tables
+import logging
+
+
+def test_dl1writer(tmpdir: Path):
+    """
+    Check that we can write DL1 files
+
+    Parameters
+    ----------
+    tmpdir :
+        temp directory fixture
+    """
+
+    output_path = Path(tmpdir / "events.dl1.h5")
+    source = event_source(
+        get_dataset_path("gamma_test_large.simtel.gz"),
```

Comment:
you should be able to use Prod3B files now

Reply:
yes, but that's for another PR - gamma_test_large will be prod 3 (or 5) in the future once we have the ability to switch between test datasets.

Reply:
I have tested this with prod3b externally, however, and it's fine.

Reply:
Or do you mean we have a prod3b test file already?

Reply:
yes, there are 4 `gamma_*_prod3b_test.simtel.gz` files in ctapipe extra (and on the download server)

Reply:
pointlike La Palma, Point Like paranal, diffuse la palma, divergent la palma

Reply:
yes I added prod3b test files sometime ago in ctapipe-extra

Reply:
Ok, will switch to one of those.

Reply:
now uses the prod3b file, and I updated ctapipe-extra's dependency to require 0.3.1 (since 0.3 didn't have that file), I also created and uploaded ctapipe-extra-0.3.1, since it wasn't in conda